holodeck/command.py

```diff
@@ -78,6 +78,47 @@ def to_json(self):
         return to_return
 
 
+class CommandCenter(object):
+
+    def __init__(self, client):
+        self._client = client
+
+        # Set up command buffer
+        self._command_bool_ptr = self._client.malloc("command_bool", [1], np.bool)
+        megabyte = 1048576  # This is the size of the command buffer that Holodeck expects/will read.
+        self._command_buffer_ptr = self._client.malloc("command_buffer", [megabyte], np.byte)
+        self._commands = CommandsGroup()
+        self._should_write_to_command_buffer = False
+
+    def clear(self):
+        self._commands.clear()
+
+    def handle_buffer(self):
+        """Checks if we should write to the command buffer, writes all of the queued commands to the buffer, and then
+        clears the contents of the self._commands list"""
+        if self._should_write_to_command_buffer:
+            self._write_to_command_buffer(self._commands.to_json())
+            self._should_write_to_command_buffer = False
+            self._commands.clear()
+
+    def enqueue_command(self, command_to_send):
+        self._should_write_to_command_buffer = True
+        self._commands.add_command(command_to_send)
+
+    def _write_to_command_buffer(self, to_write):
+        """Write input to the command buffer.  Reformat input string to the correct format.
+
+        Args:
+            to_write (str): The string to write to the command buffer.
+        """
+        # TODO(mitch): Handle the edge case of writing too much data to the buffer.
```

Comment:
Every time I see this I think we need to get rid of it and or make a work item for it.